requirements/runtime.txt

```diff
@@ -0,0 +1,9 @@
+fire
+gradio
```

Comment:
I think it's better to put gradio into serve.txt, @AllentDan what's your opinion?

Reply:
Agree.